go-controller/pkg/cluster/node.go

```diff
@@ -107,6 +108,7 @@ func (cluster *OvnClusterController) StartClusterNode(name string) error {
 	confJSON := fmt.Sprintf("{\"name\":\"ovn-kubernetes\", \"type\":\"%s\"}", config.CNI.Plugin)
 	_, err = f.Write([]byte(confJSON))
 
+	cluster.watchNamespaceUpdate(node, subnet.String())
```

Comment:
Won't we just return from this function and won't ovnkube node-init simply exit? I think we need to wait forever if -ha is provided and 'node != "" ' in cmd/ovnkube/ovnkube.go

Reply:
Done